packages/react-components/src/components/ControlBarButton.tsx

```diff
@@ -133,19 +133,19 @@ export const ControlBarButton = (props: ControlBarButtonProps): JSX.Element => {
       ? props?.strings?.tooltipOnContent
       : props?.strings?.tooltipOffContent);
 
+  const splitButtonAriaLabel = props.splitButtonAriaLabel ? tooltipContent + props.splitButtonAriaLabel : undefined;
```

Comment:
This shouldn't combine the tooltipContent and the aria label, instead just have the aria label be `Choose camera and camera options`.

Reply:
e.g. while the video is loading and the button is disabled the tooltip is `video is loading`